lms/djangoapps/ccx/tests/test_overrides.py

```diff
@@ -66,7 +67,7 @@ def setUp(self):
         OverrideFieldData.provider_classes = None
         for block in iter_blocks(course):
             block._field_data = OverrideFieldData.wrap(   # pylint: disable=protected-access
```

Comment:
There should be a test to see what happen if ccx_enable is false, i mean check OverrideFieldData is wrap or not